modules/ztp-du-host-bios-requirements.adoc

```diff
@@ -0,0 +1,87 @@
+// CNF-1500 ZTP
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="configuring_bios_for_du_hosts_{context}"]
+= Configuring BIOS for distributed unit bare-metal hosts
+
+Distributed unit (DU) hosts require the BIOS to be configured before the host can be provisioned. The BIOS configuration is dependent on the specific hardware that runs your DUs and the particular requirements of your installation.
```

Comment:
Not sure if this is covered elsewhere.  Automated setting of the BIOS is not included in this release.  It should be listed as a pre-requisite.

Reply:
Added this as an admonition in this section.